conda/environments/rapids_triton_dev.yml

```
@@ -4,7 +4,7 @@ channels:
   - conda-forge
 dependencies:
   - ccache
-  - cmake>=3.21,<3.23
+  - cmake>=3.23,<3.24
```

```suggestion
  - cmake>=3.23.1
```
With latest version of rapids-cmake, that should be sufficient. No need to limit the highest allowed version.